node/clients/storefrontPermissions.ts

```diff
@@ -22,35 +21,49 @@ export default class StorefrontPermissions extends AppGraphQLClient {
     super('vtex.storefront-permissions@1.x', ctx, options)
   }
 
-  public checkUserPermission = async (app?: string): Promise<any> => {
+  private getTokenToHeader = (ctx: IOContext) => {
+    return {
+      VtexIdclientAutCookie:
+        ctx.storeUserAuthToken ?? ctx.adminUserAuthToken ?? ctx.authToken,
+    }
+  }
+
+  public checkUserPermission = async (): Promise<any> => {
     return this.graphql.query(
       {
         extensions: {
           persistedQuery: {
             provider: 'vtex.storefront-permissions@1.x',
-            sender: app ?? 'vtex.b2b-organizations@0.x',
+            sender: 'vtex.b2b-organizations-graphql@0.x',
           },
         },
         query: getPermission,
         variables: {},
       },
-      {}
+      {
+        headers: this.getTokenToHeader(this.context),
+      }
     )
   }
 
   public getOrganizationsByEmail = async (email: string): Promise<any> => {
-    return this.graphql.query({
-      extensions: {
-        persistedQuery: {
-          provider: 'vtex.storefront-permissions@1.x',
-          sender: 'vtex.b2b-organizations@0.x',
+    return this.graphql.query(
+      {
+        extensions: {
+          persistedQuery: {
+            provider: 'vtex.storefront-permissions@1.x',
+            sender: 'vtex.b2b-organizations-graphql@0.x',
+          },
+        },
```

Comment:
as this is basically the same for all queries, is there a way to parametrize this? so in the future we can change parameters (e.g. the app version) in a single place if needed?